### DISCUSSION TITLE
Can i use this package without gorouter?

### DISCUSSION THREAD

**Discussion by andikatp** [fujidaiti/smooth_sheets#142] [Q&A] [answered]
im a newbie, right now i use getx, since its already have it own routing system, i dont want to have another routing anymore, can i do it without gorouter? thanks

**Comment by fujidaiti** [comment, marked as answer]
Hi @andikatp,

>can i do it without gorouter?

Of course yes! this package has no code that depends on go_router. I don't know much about the getx's navigation system, but this package can work fine if it supports either imperative ways (like `Navigator.push`) or declarative ways (generally called Navigator 2.0).

**Reply by andikatp** [reply]
oh okay thank you i saw the cookbook about ai_playlist_generator that use go router, thats why i thought i need it, thank you for the answer
